examples/discrete_sac/config.yaml

```diff
@@ -1,37 +1,52 @@
-# Logger
-logger: wandb
-exp_name: discrete_sac
-record_interval: 1
-mode: online
 
-# Environment
-env_name: CartPole-v1
-frame_skip: 1
-from_pixels: false
-reward_scaling: 1.0
-init_env_steps: 1000
-seed: 42
+# task and env
+env:
+  name: CartPole-v1
+  task: ""
+  exp_name: ${env.name}_DiscreteSAC
+  library: gym
+  seed: 42
+  max_episode_steps: 500
 
-# Collector
-env_per_collector: 1
-max_frames_per_traj: 500
-total_frames: 1000000
-init_random_frames: 5000
-frames_per_batch: 500 # 500 * env_per_collector
+# collector
+collector:
+  total_frames: 25000
+  init_random_frames: 1000
+  init_env_steps: 1000
+  frames_per_batch: 500
+  reset_at_each_iter: False
+  device: cpu
```

Comment:
why is the collector on cpu and the model on cuda?

Reply:
changed it to cuda :)